Level 1 Layout Expansion

PiersSK opened the pull request:
Loads of new fancy environment! Game should still be completable in all routes. Currently there's no legitimate way into the Engine Room.

PiersSK committed:
feat(code): adds walls for new rooms

PiersSK committed:
feat(code): adds draft ensuite

PiersSK committed:
feat(code): adds ensuite to LQ2

PiersSK committed:
feat(code): health room kitchen

PiersSK committed:
feat(code): furnishes rest of Health

PiersSK committed:
fix(code): reenabled UI

PiersSK committed:
feat(code): first draft of new server room

PiersSK committed:
feat(code): adds working area of server room

PiersSK committed:
feat(code): tidies server room hierarchy and adds doors

PiersSK committed:
feat(code): adds doors, utility and tidies scene

PiersSK committed:
feat(code): doomsday room update

PiersSK committed:
feat(code): adds engine room

PiersSK committed:
fix(code): fixed strange artefact on middle floor

PiersSK committed:
feat(code): adds cosy/horror bits to health

PiersSK committed:
Merge branch 'develop' into feat/level-layout-expansion

PiersSK committed:
fix(code): corrects merge conflict oddities

- fixes duplicate ids
- fixes objects with missing info

PiersSK committed:
fix(code): fix strange colliders with paper stacks

PiersSK committed:
Merge branch 'develop' into feat/level-layout-expansion

PiersSK committed:
fix(code): corrects prefabs affected by dev merge

JamesYJ95 reviewed (changes requested):
Spotted a couple bugs here:
1. Tidy quarters task - Task Pad is counting as a tidied up object when the player first picks it up, which can cause the player to either never complete the Tidy Quarters Task, or complete it too early.
2. Upload fleet data task - Getting an UnassignedReferenceException on the fire button for commanderVLSource

OrcaOfTime committed:
feat(code): Fixed taskpad clean up task bug

Task pad was inheriting from cleanableInteractable for the 'setActive(false)' function so it would disappear from the table when picked up however this ticked the clean up counter for the clean quarters task mistakenly. Have change inherited class and added that one line to the taskpad interactable

OrcaOfTime committed:
feat(scene): Added audio back to CommandVLSource

Was missing in editor and causing a Null ref exception in script code
